relayer/core/relay.go

```diff
@@ -52,16 +54,34 @@ func NewRelay() (*Relay, error) {
 		return nil, err
 	}
 
-	ethChain, err := ethereum.NewChain(&config.Eth)
+	// TODO: integrate with config
```

Comment:
do in this PR plz as it is related to general plumbing/structure of relayer

Reply:
Updated in https://github.com/Snowfork/polkadot-ethereum/pull/314/commits/ec27500ba0c88d0268ac6faaa85f59f2ab3dbffd